Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -178,6 +190,9 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
 
     // MARK: Private Methods / Properties
 
+    private var bottomConstraint: NSLayoutConstraint?
```

Comment:
Probably better to group all of your instance variables together, near the top preferably (but under the public stuff).